platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -9,12 +9,12 @@ dependencies {
     compile 'com.jakewharton.timber:timber:4.3.1'
```

Comment:
strange I thought I had moved all of these to dependencies.gradle? if not could you pick this up with this PR?